velox/functions/sparksql/Map.cpp

```diff
@@ -18,8 +18,8 @@
 #include "velox/expression/VectorFunction.h"
 #include "velox/vector/VectorTypeUtils.h"
 
-// The description of the map function in Spark
-// https://kontext.tech/article/586/spark-sql-map-functions
```

Comment:
Shall we keep the previous documentation? And it looks like these comments belong to the 'class MapFunction` at L61.